internal/integration/api/reset.go

```diff
@@ -218,9 +218,9 @@ func (suite *ResetSuite) TestResetWithSpecState() {
 		suite.T().Skip("cluster doesn't support reboot (and reset)")
 	}
 
-	if suite.Capabilities().TrustedBoot {
-		// this is because with trusted boot the talos.config is only applied and cannot be passed as kernel args
-		suite.T().Skip("cluster has trusted boot enabled, doesn't support reset")
+	if suite.TrustedBoot {
```

Comment:
I think the previous made more sense - if we detect secure boot, we skip the tests, vs. that we were asked to verify that trusted boot is enabled.

Reply:
didn't get that

Reply:
in one of the previous iterations of this PR, you had `Capabilities().TrustedBoot` driven by the `SecurityState` resource, I remember reviewing it.

what I mean that this test should be skipped if the cluster is running in trusted boot mode, no matter if the flag `-test.trustedboot` was specified or not.